src/test/java/org/folio/rest/impl/HelperUtilsTest.java

```diff
@@ -36,6 +36,8 @@ public class HelperUtilsTest extends TestBase {
   private static final String RECEIVING_HISTORY_ENDPOINT = "/orders-storage/receiving-history";
 
   @Test
+  @Disabled("disabled due to incompatibility of jdk17 + jmockit + jacoco"
+    + "remove annotation and check with command: 'mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install' ")
```

Comment:
Tests will be enabled again after migrating from JMockit to Mockito: https://issues.folio.org/browse/MODORDSTOR-355